OpenCover.UI.TestDiscoverer.Tests/OpenCover.UI.TestDiscoverer.Tests.csproj

```diff
@@ -59,12 +61,28 @@
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
+    <Reference Include="xunit.abstractions">
+      <HintPath>..\packages\xunit.abstractions.2.0.0\lib\net35\xunit.abstractions.dll</HintPath>
+    </Reference>
+    <Reference Include="xunit.assert, Version=2.1.0.3179, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
+      <HintPath>..\packages\xunit.assert.2.1.0\lib\dotnet\xunit.assert.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="xunit.core, Version=2.1.0.3179, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
+      <HintPath>..\packages\xunit.extensibility.core.2.1.0\lib\dotnet\xunit.core.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="xunit.execution.desktop, Version=2.1.0.3179, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
+      <HintPath>..\packages\xunit.extensibility.execution.2.1.0\lib\net45\xunit.execution.desktop.dll</HintPath>
+      <Private>True</Private>
```

Comment:
@joshlrogers Googling the appveyor error message (missing dlls) seems to suggest you need to add these references to [OpenCover.UI.csproj](https://github.com/joshlrogers/OpenCover.UI/blob/master/OpenCover.UI/OpenCover.UI.csproj)

Or maybe better alternatives:
  * Somehow tell appveyor to ignore discovering `OpenCover.UI.TestDiscoverer.exe` as a container since the tests are in `OpenCover.UI.TestDiscoverer.Tests.dll` or
  * Stick to NUnit test attributes rather than using [Fact] in [XunitDiscovererTests.cs](https://github.com/joshlrogers/OpenCover.UI/tree/master/OpenCover.UI.TestDiscoverer.Tests/Xunit)

references:
http://help.appveyor.com/discussions/problems/2508-xunit-2-tests-discovered-but-runner-not-found
http://help.appveyor.com/discussions/problems/677-xunit-20-support